2024-10-26 01:20:33 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [99] - RabbitMQ consumer cancelled. --> amq.ctag-O_FVZJklM-HJ4db70PK5ww
2024-10-26 01:20:33 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [99] - RabbitMQ consumer shutdown. --> Unexpected Exception: Unable to read data from the transport connection: 远程主机强迫关闭了一个现有的连接。.
2024-10-26 01:20:34 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [26] - RabbitMQ consumer cancelled. --> amq.ctag-Xf_MI-mRke5PvrKnQzVeng
2024-10-26 01:20:37 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [26] - RabbitMQ consumer shutdown. --> Unexpected Exception: Unable to read data from the transport connection: 远程主机强迫关闭了一个现有的连接。.
2024-10-26 01:20:38 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [26] - RabbitMQ consumer cancelled. --> amq.ctag--gvuL0TZd0yZ-OiNinhcHQ
2024-10-26 01:20:38 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [26] - RabbitMQ consumer shutdown. --> Unexpected Exception: Unable to read data from the transport connection: 远程主机强迫关闭了一个现有的连接。.
2024-10-26 01:20:38 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [56] - RabbitMQ consumer registered. --> amq.ctag-dJPtPg-cxjvOZtkjbbZioQ
2024-10-26 01:20:39 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [26] - RabbitMQ consumer registered. --> amq.ctag-O_FVZJklM-HJ4db70PK5ww
2024-10-26 01:20:39 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [58] - RabbitMQ consumer registered. --> amq.ctag-Xf_MI-mRke5PvrKnQzVeng
2024-10-26 01:20:39 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [47] - RabbitMQ consumer registered. --> amq.ctag--gvuL0TZd0yZ-OiNinhcHQ
2024-10-26 02:04:40 [WRN] [Microsoft.AspNetCore.Http.ResponseCookies] [14928] [47] - The cookie 'XSRF-TOKEN' has set 'SameSite=None' and must also set 'Secure'.
2024-10-26 02:04:48 [WRN] [Volo.Abp.AspNetCore.Mvc.AspNetCoreApiDescriptionModelProvider] [14928] [43] - Controller 'Notification' contains more than one action with name 'SendAsyncByInput' for module 'notifications'. Ignored: System.Threading.Tasks.Task SendAsync(ZQH.Abp.Notifications.NotificationTemplateSendDto)
2024-10-26 03:22:32 [WRN] [Microsoft.AspNetCore.Http.ResponseCookies] [14928] [52] - The cookie 'XSRF-TOKEN' has set 'SameSite=None' and must also set 'Secure'.
2024-10-26 03:22:40 [WRN] [Volo.Abp.AspNetCore.Mvc.AspNetCoreApiDescriptionModelProvider] [14928] [52] - Controller 'Notification' contains more than one action with name 'SendAsyncByInput' for module 'notifications'. Ignored: System.Threading.Tasks.Task SendAsync(ZQH.Abp.Notifications.NotificationTemplateSendDto)
2024-10-26 03:22:49 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [52] - Executing subscriber method 'IdentitySessionCacheItemSynchronizer.HandleEventAsync' on group 'MessageService.v1'
2024-10-26 03:22:49 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [98] - Executed subscriber method 'IdentitySessionCacheItemSynchronizer.HandleEventAsync' on group 'MessageService.v1' with instance 'zqhT16' in 8.2732ms
2024-10-26 03:22:51 [WRN] [Microsoft.AspNetCore.Http.ResponseCookies] [14928] [94] - The cookie 'XSRF-TOKEN' has set 'SameSite=None' and must also set 'Secure'.
2024-10-26 03:24:26 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [36] - Executing subscriber method 'IdentitySessionRevokeEventHandler.HandleEventAsync' on group 'MessageService.IdentitySessionRevokeEventHandler.v1'
2024-10-26 03:24:26 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [73] - Executing subscriber method 'IdentitySessionCacheItemSynchronizer.HandleEventAsync' on group 'MessageService.v1'
2024-10-26 03:24:26 [INF] [DotNetCore.CAP.RabbitMQ.RabbitMQTransport] [14928] [52] - CAP message 'abp.realtime.notifications' published, internal id '4001002700606074899'
2024-10-26 03:24:26 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [154] - Executed subscriber method 'IdentitySessionRevokeEventHandler.HandleEventAsync' on group 'MessageService.IdentitySessionRevokeEventHandler.v1' with instance 'zqhT16' in 51.5606ms
2024-10-26 03:24:26 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [151] - Executed subscriber method 'IdentitySessionCacheItemSynchronizer.HandleEventAsync' on group 'MessageService.v1' with instance 'zqhT16' in 34.6127ms
2024-10-26 03:24:26 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [73] - Executing subscriber method 'NotificationEventHandler.HandleEventAsync' on group 'MessageService.v1'
2024-10-26 03:24:27 [WRN] [ZQH.Abp.Notifications.WeChat.Work.WeChatWorkNotificationPublishProvider] [14928] [151] - WeChat.Work cannot push messages because the feature WeChat.Work.Message.Enable is not enabled
2024-10-26 03:24:27 [WRN] [ZQH.Abp.Notifications.WeChat.MiniProgram.WeChatMiniProgramNotificationPublishProvider] [14928] [151] - Wechat weapp template id be empty, can not send notification!
2024-10-26 03:24:27 [WRN] [ZQH.Abp.Notifications.WxPusher.WxPusherNotificationPublishProvider] [14928] [151] - WxPusher cannot push messages because the feature WxPusher.Message.Enable is not enabled
2024-10-26 03:24:27 [WRN] [ZQH.Abp.Notifications.Emailing.EmailingNotificationPublishProvider] [14928] [52] - The subscriber did not confirm the email address and could not send email notifications!
2024-10-26 03:24:27 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [151] - Executed subscriber method 'NotificationEventHandler.HandleEventAsync' on group 'MessageService.v1' with instance 'zqhT16' in 1177.9619ms
2024-10-26 03:24:32 [WRN] [Microsoft.AspNetCore.Http.ResponseCookies] [14928] [151] - The cookie 'XSRF-TOKEN' has set 'SameSite=None' and must also set 'Secure'.
2024-10-26 03:24:34 [WRN] [Volo.Abp.AspNetCore.Mvc.AspNetCoreApiDescriptionModelProvider] [14928] [52] - Controller 'Notification' contains more than one action with name 'SendAsyncByInput' for module 'notifications'. Ignored: System.Threading.Tasks.Task SendAsync(ZQH.Abp.Notifications.NotificationTemplateSendDto)
2024-10-26 03:24:43 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [151] - Executing subscriber method 'IdentitySessionCacheItemSynchronizer.HandleEventAsync' on group 'MessageService.v1'
2024-10-26 03:24:43 [INF] [DotNetCore.CAP.Internal.SubscribeExecutor] [14928] [151] - Executed subscriber method 'IdentitySessionCacheItemSynchronizer.HandleEventAsync' on group 'MessageService.v1' with instance 'zqhT16' in 4.5114ms
2024-10-26 03:24:44 [WRN] [Microsoft.AspNetCore.Http.ResponseCookies] [14928] [52] - The cookie 'XSRF-TOKEN' has set 'SameSite=None' and must also set 'Secure'.
2024-10-26 04:00:40 [WRN] [Volo.Abp.Caching.DistributedCache] [14928] [34] - The operation was canceled.
System.OperationCanceledException: The operation was canceled.
   at System.Threading.CancellationToken.ThrowOperationCanceledException()
   at Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache.GetAsync(String key, CancellationToken token)
   at Volo.Abp.Caching.DistributedCache`2.GetAsync(TCacheKey key, Nullable`1 hideErrors, Boolean considerUow, CancellationToken token)
2024-10-26 04:00:40 [WRN] [Volo.Abp.Caching.DistributedCache] [14928] [7] - The operation was canceled.
System.OperationCanceledException: The operation was canceled.
   at System.Threading.CancellationToken.ThrowOperationCanceledException()
   at Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache.GetAsync(String key, CancellationToken token)
   at Volo.Abp.Caching.DistributedCache`2.GetAsync(TCacheKey key, Nullable`1 hideErrors, Boolean considerUow, CancellationToken token)
2024-10-26 04:00:40 [ERR] [Microsoft.AspNetCore.SignalR.HubConnectionHandler] [14928] [34] - Error when dispatching 'OnDisconnectedAsync' on hub.
System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at Volo.Abp.Threading.SemaphoreSlimExtensions.LockAsync(SemaphoreSlim semaphoreSlim, CancellationToken cancellationToken)
   at Volo.Abp.Caching.DistributedCache`2.GetOrAddAsync(TCacheKey key, Func`1 factory, Func`1 optionsFactory, Nullable`1 hideErrors, Boolean considerUow, CancellationToken token)
   at Volo.Abp.Identity.IdentityDynamicClaimsPrincipalContributorCache.GetAsync(Guid userId, Nullable`1 tenantId)
   at Volo.Abp.Identity.IdentityDynamicClaimsPrincipalContributor.ContributeAsync(AbpClaimsPrincipalContributorContext context)
   at Volo.Abp.Security.Claims.AbpClaimsPrincipalFactory.InternalCreateAsync(AbpClaimsPrincipalFactoryOptions options, ClaimsPrincipal existsClaimsPrincipal, Boolean isDynamic)
   at Volo.Abp.Security.Claims.AbpClaimsPrincipalFactory.CreateDynamicAsync(ClaimsPrincipal existsClaimsPrincipal)
   at Volo.Abp.AspNetCore.SignalR.Authentication.AbpAuthenticationHubFilter.HandleDynamicClaimsPrincipalAsync(ClaimsPrincipal claimsPrincipal, IServiceProvider serviceProvider, HubCallerContext hubCallerContext, Boolean skipCheckDynamicClaimsInterval)
   at Volo.Abp.AspNetCore.SignalR.Authentication.AbpAuthenticationHubFilter.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.OnDisconnectedAsync(HubConnectionContext connection, Exception exception)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.OnDisconnectedAsync(HubConnectionContext connection, Exception exception)
   at Microsoft.AspNetCore.SignalR.HubConnectionHandler`1.HubOnDisconnectedAsync(HubConnectionContext connection, Exception exception)
2024-10-26 04:00:40 [ERR] [Microsoft.AspNetCore.SignalR.HubConnectionHandler] [14928] [7] - Error when dispatching 'OnDisconnectedAsync' on hub.
System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at Volo.Abp.Threading.SemaphoreSlimExtensions.LockAsync(SemaphoreSlim semaphoreSlim, CancellationToken cancellationToken)
   at Volo.Abp.Caching.DistributedCache`2.GetOrAddAsync(TCacheKey key, Func`1 factory, Func`1 optionsFactory, Nullable`1 hideErrors, Boolean considerUow, CancellationToken token)
   at Volo.Abp.Identity.IdentityDynamicClaimsPrincipalContributorCache.GetAsync(Guid userId, Nullable`1 tenantId)
   at Volo.Abp.Identity.IdentityDynamicClaimsPrincipalContributor.ContributeAsync(AbpClaimsPrincipalContributorContext context)
   at Volo.Abp.Security.Claims.AbpClaimsPrincipalFactory.InternalCreateAsync(AbpClaimsPrincipalFactoryOptions options, ClaimsPrincipal existsClaimsPrincipal, Boolean isDynamic)
   at Volo.Abp.Security.Claims.AbpClaimsPrincipalFactory.CreateDynamicAsync(ClaimsPrincipal existsClaimsPrincipal)
   at Volo.Abp.AspNetCore.SignalR.Authentication.AbpAuthenticationHubFilter.HandleDynamicClaimsPrincipalAsync(ClaimsPrincipal claimsPrincipal, IServiceProvider serviceProvider, HubCallerContext hubCallerContext, Boolean skipCheckDynamicClaimsInterval)
   at Volo.Abp.AspNetCore.SignalR.Authentication.AbpAuthenticationHubFilter.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.OnDisconnectedAsync(HubConnectionContext connection, Exception exception)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.OnDisconnectedAsync(HubConnectionContext connection, Exception exception)
   at Microsoft.AspNetCore.SignalR.HubConnectionHandler`1.HubOnDisconnectedAsync(HubConnectionContext connection, Exception exception)
2024-10-26 04:00:40 [ERR] [Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionManager] [14928] [67] - Failed disposing connection w_OBfZzH76VvMstAYF1Bcw.
System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at Volo.Abp.Threading.SemaphoreSlimExtensions.LockAsync(SemaphoreSlim semaphoreSlim, CancellationToken cancellationToken)
   at Volo.Abp.Caching.DistributedCache`2.GetOrAddAsync(TCacheKey key, Func`1 factory, Func`1 optionsFactory, Nullable`1 hideErrors, Boolean considerUow, CancellationToken token)
   at Volo.Abp.Identity.IdentityDynamicClaimsPrincipalContributorCache.GetAsync(Guid userId, Nullable`1 tenantId)
   at Volo.Abp.Identity.IdentityDynamicClaimsPrincipalContributor.ContributeAsync(AbpClaimsPrincipalContributorContext context)
   at Volo.Abp.Security.Claims.AbpClaimsPrincipalFactory.InternalCreateAsync(AbpClaimsPrincipalFactoryOptions options, ClaimsPrincipal existsClaimsPrincipal, Boolean isDynamic)
   at Volo.Abp.Security.Claims.AbpClaimsPrincipalFactory.CreateDynamicAsync(ClaimsPrincipal existsClaimsPrincipal)
   at Volo.Abp.AspNetCore.SignalR.Authentication.AbpAuthenticationHubFilter.HandleDynamicClaimsPrincipalAsync(ClaimsPrincipal claimsPrincipal, IServiceProvider serviceProvider, HubCallerContext hubCallerContext, Boolean skipCheckDynamicClaimsInterval)
   at Volo.Abp.AspNetCore.SignalR.Authentication.AbpAuthenticationHubFilter.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.HubFilterFactory.OnDisconnectedAsync(HubLifetimeContext context, Exception exception, Func`3 next)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.OnDisconnectedAsync(HubConnectionContext connection, Exception exception)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.OnDisconnectedAsync(HubConnectionContext connection, Exception exception)
   at Microsoft.AspNetCore.SignalR.HubConnectionHandler`1.HubOnDisconnectedAsync(HubConnectionContext connection, Exception exception)
   at Microsoft.AspNetCore.SignalR.HubConnectionHandler`1.RunHubAsync(HubConnectionContext connection)
   at Microsoft.AspNetCore.SignalR.HubConnectionHandler`1.OnConnectedAsync(ConnectionContext connection)
   at Microsoft.AspNetCore.SignalR.HubConnectionHandler`1.OnConnectedAsync(ConnectionContext connection)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionContext.ExecuteApplication(ConnectionDelegate connectionDelegate)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionContext.WaitOnTasks(Task applicationTask, Task transportTask, Boolean closeGracefully)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionContext.DisposeAsync(Boolean closeGracefully)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionManager.DisposeAndRemoveAsync(HttpConnectionContext connection, Boolean closeGracefully, HttpConnectionStopStatus status)
2024-10-26 06:06:11 [ERR] [Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionManager] [14928] [46] - Failed disposing connection yzq9RQ7Jgx5A8HBmp3StZQ.
StackExchange.Redis.RedisTimeoutException: Timeout awaiting response (outbound=0KiB, inbound=0KiB, 5860ms elapsed, timeout is 5000ms), command=UNSUBSCRIBE, next: UNSUBSCRIBE ZQH.Abp.IM.SignalR.Hubs.MessagesHub:connection:yzq9RQ7Jgx5A8HBmp3StZQ, inst: 0, qu: 0, qs: 2, aw: False, bw: Inactive, rs: ReadAsync, ws: Idle, in: 0, in-pipe: 0, out-pipe: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 56, serverEndpoint: 127.0.0.1:6379, conn-sec: 84507.38, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: ZQHT16(SE.Redis-v2.7.4.20928), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=10,Free=32757,Min=16,Max=32767), POOL: (Threads=11,QueuedItems=1,CompletedItems=267884,Timers=24), v: 2.7.4.20928 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
   at Microsoft.AspNetCore.SignalR.StackExchangeRedis.Internal.RedisSubscriptionManager.RemoveSubscriptionAsync(String id, HubConnectionContext connection, Object state, Func`3 unsubscribeMethod)
   at Microsoft.AspNetCore.SignalR.HubConnectionHandler`1.OnConnectedAsync(ConnectionContext connection)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionContext.ExecuteApplication(ConnectionDelegate connectionDelegate)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionContext.WaitOnTasks(Task applicationTask, Task transportTask, Boolean closeGracefully)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionContext.DisposeAsync(Boolean closeGracefully)
   at Microsoft.AspNetCore.Http.Connections.Internal.HttpConnectionManager.DisposeAndRemoveAsync(HttpConnectionContext connection, Boolean closeGracefully, HttpConnectionStopStatus status)
2024-10-26 06:06:04 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [70] - RabbitMQ consumer cancelled. --> amq.ctag-dJPtPg-cxjvOZtkjbbZioQ
2024-10-26 09:47:53 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [70] - RabbitMQ consumer shutdown. --> Unexpected Exception: Unable to read data from the transport connection: 远程主机强迫关闭了一个现有的连接。.
2024-10-26 09:47:53 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [64] - RabbitMQ consumer cancelled. --> amq.ctag-O_FVZJklM-HJ4db70PK5ww
2024-10-26 09:47:53 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [64] - RabbitMQ consumer shutdown. --> Unexpected Exception: Unable to read data from the transport connection: 远程主机强迫关闭了一个现有的连接。.
2024-10-26 09:47:53 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [67] - RabbitMQ consumer cancelled. --> amq.ctag-Xf_MI-mRke5PvrKnQzVeng
2024-10-26 09:47:53 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [67] - RabbitMQ consumer shutdown. --> Unexpected Exception: Unable to read data from the transport connection: 远程主机强迫关闭了一个现有的连接。.
2024-10-26 09:47:53 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [70] - RabbitMQ consumer cancelled. --> amq.ctag--gvuL0TZd0yZ-OiNinhcHQ
2024-10-26 09:47:53 [WRN] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [70] - RabbitMQ consumer shutdown. --> Unexpected Exception: Unable to read data from the transport connection: 远程主机强迫关闭了一个现有的连接。.
2024-10-26 09:47:53 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [70] - RabbitMQ consumer registered. --> amq.ctag-dJPtPg-cxjvOZtkjbbZioQ
2024-10-26 09:47:53 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [46] - RabbitMQ consumer registered. --> amq.ctag-O_FVZJklM-HJ4db70PK5ww
2024-10-26 09:47:53 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [77] - RabbitMQ consumer registered. --> amq.ctag-Xf_MI-mRke5PvrKnQzVeng
2024-10-26 09:47:53 [INF] [DotNetCore.CAP.Internal.ConsumerRegister] [14928] [67] - RabbitMQ consumer registered. --> amq.ctag--gvuL0TZd0yZ-OiNinhcHQ
